swarm/README.md

```diff
@@ -0,0 +1,53 @@
+Mini-broker Swarm deployment for isolated containerized filters
+--------------------------------------------------------------
+
+These are Swarm deployment scripts for a version of the
+mini-broker running a one-container-per-filter setup.
+This design is updated from the design described in DMTN-081
+[https://dmtn-028.lsst.io](https://dmtn-028.lsst.io), which
+runs multiple filters in the same container.
+To deploy a prototype of the full mini-broker with 100 consumers
+(end users) and 100 individual filters each in their own container,
+run the `deploy_all.sh` script.
+This will deploy the following setup on a total of 12 nodes (one manager and
+11 worker nodes):
+
+* A main Kafka/Zookeeper hub on the Swarm manager node that receives the full alert stream and runs by itself on the manager node
+* A single alert producer running on its own separate node that sends the full stream of alerts to the main Kafka/Zookeeper hub
+* Five downstream Kafka/Zookeepers for feeding the filterers, which mirror the full stream and run each on a separate node
+* 100 stream filterer containers split into 20 on each of the five Kafka/Zookeeper mirror nodes, which read the full stream and send filtered alert streams back to the co-located Kafka mirror
+* 100 consumers of the filtered streams running as 20 separate containers on five separate nodes and read from the Kafka mirrors
+
+Required are `consumernodes.txt`, `filternodes.txt`, and `sendernodes.txt`,
+each with the Swarm IDs of the nodes on which to run the different components
+on separate lines.  
+The consumer and filter nodes files should each have the IDs of five unique nodes,
+so that each node will run 20 consumer or 20 filter containers, for a total
+of 100 consumers and 100 filters.
+The sender node file needs one Swarm ID on which to place a single
+alert producer sending 10,000 alerts per visit (every 39 seconds).
```

Comment:
Could you put this in the context of LSST? Is the requirement 10,000 alerts per visit every 39 seconds, or where do these numbers come from?

Reply:
Thanks, I put in a reference to the requirement.